app/posts/manage-teacher-training-applications/2020-05-18-access-control-history.md

```diff
@@ -0,0 +1,551 @@
+
+##### Catholic Teaching School Alliance (CTSA)
+
+A School Direct with 4 different HEI accredited bodies.
+* [Session notes](https://docs.google.com/document/d/1hSYC9knyDeW1LsUlenZeil9tkoeoIPRZTc7v5YZeL4Y/edit)
+* [Process map](https://drive.google.com/drive/folders/1p4Ap6BwUsEi4X_5xuqY1wp2QgY6IUHf5)
+
+__Key access control findings:__
+* Each accredited body HEI has a slightly different process that they want to be followed for managing applications for courses they ratify
+* CTSA are very much in control of driving the process and have to be mindful of which HEI an application is for
+* CTSA are able to move candidates from a course ratified by one HEI to a course ratified by another HEI
+* CTSA use an external assessor for sifting applications. It would be very useful if this person could have access to Apply, but be restricted from any decision-making actions
+* The HEIs need to see the safeguarding information, CTSA do not.
+
+##### Keele and North Staffordshire Teacher Education (KNSTE) 
+
+An accredited body and SCITT, ratifying courses for 12 partner schools, of which 4 are Schools Direct, some of whom are ratified  by other accredited bodies. KNSTE also run their own courses.
+* [Session notes](https://docs.google.com/document/d/1QOGasJVs06b4NMhBWnPj9bIrhPL_P5ggf0t_N1F_2yo/edit)
+
+__Key access control findings:__
+* Each school has an agreement with KNSTE SCITT on ways of working
+* People from the SCITT and from the schools are involved in decision-making, but most of the time KNSTE are uploading decisions into UCAS. So there are users who only need to view applications.
+* However, given that the Schools Direct are also working with other accredited bodies, they will have their own additional ways of working agreements. These may be quite different to how they work with KNSTE
+* The director of the SCITT handles anything to do with safeguarding
+
+##### Potential access models
+
+[Spreadsheet modeling how access permissions could work for these organisations](https://docs.google.com/spreadsheets/d/19D1PUSff78vevA8uuGtVJsnsIz7di_x3CNvbupzxNnc/edit#gid=0)
+
+#### Organisation types & links
+
+We developed a set of organisational diagrams to explain the different set ups within organisations. These were developed with evidence from qualitative research sessions, through policy documentation and historical research across teams in DfE. 
+
+These diagrams show the organisational structures, but do not explain what different providers within the organisation are responsible for.
+
+[Organisational diagrams folder](https://drive.google.com/drive/u/0/folders/1p4Ap6BwUsEi4X_5xuqY1wp2QgY6IUHf5)
+
+#### Safeguarding research
+
+During research, we identified a candidate need to explain any criminal convictions:
+
+>__As a__ candidate
```

Comment:
Should this use the user needs macro and link this to a common user need page?

Reply:
I couldn't work out how that worked. I only got screenshots working by chance ;)

The challenge really is that the needs are best kept and managed in the Trello board. I only see 12 actual needs in the history, which is obviously not the complete set, so I guess it's whether we want to commit the effort to getting them all over here somehow

Reply:
Yeah, separate discussion. It needs to be all or nothing, else they’ll diverge. There’s a good case to be made for either option, Trello or on the design history.

Reply:
Something it would be worth discussing in the cross team UX meetings if we ever have those again